test/test-component.js

```diff
@@ -77,31 +77,20 @@ describe('Nested Component instance', function() {
   var el, childEl;
 
   context('before attached to DOM', function() {
-    beforeEach(function(done) {
+    beforeEach(function() {
       document.body.innerHTML = '';
       el = document.createElement('nested-app');
```

Comment:
Think we actually need this from below as well
```
       window.requestAnimationFrame(function() {
         childEl = el.getElementsByTagName('nested-child')[0];
         done();
       });
```
Otherwise childEl may be undefined, as seems to be happening in the CI test runs